_episodes/02-shell-basics.md

```diff
@@ -1,31 +1,418 @@
+just like the graphical interface (windows, pointer, icons, toolbars and menus)
+that you may be more familiar with.
+The difference is that rather than
+clicking on buttons, menu items and check-boxes, or entering text into graphical boxes,
+you provide instructions to the computer by
+issuing commands to it.
 
-Recommendation: for a novice audience, go through 1,2,3 in 
-the [shell-novice lesson](http://swcarpentry.github.io/shell-novice/)
+As a few examples of tasks you might do on your computer,
+consider:
 
-Also convey: 
-The shell is more than a way to look at files or folders. It's also a way to run programs.
+* Creating new files or folders
+* Navigating between folders on your computer and viewing their contents
+* Moving, renaming, deleting or copying files and folders
 
-Programs may expect data as input and may produce data as output. The shell lets us 
-control which programs to run on which inputs and what to do with the output.
+You are probably familiar with using a graphical file browser
+such as **Windows Explorer** (Windows) or **Finder** (Mac OS X)
+for doing the above tasks:
+The same tasks can be performed on the command-line,
+using a few simple commands:
 
-Maybe use a shell program like sleep or grep as an example, or another simple executable.  
+| Command 	  			| Action
+------------------------|-------------------------------------------
+| `pwd`		  			| Print working directory
+| `cd DIR` 				| Change directory to `DIR`	
+| `ls`					| List contents of current directory
+| `mkdir DIR`			| Make new empty directory `DIR`
+| `touch FILE`          | Make new empty file `FILE`
+| `cp SRC DEST`			| Copy file `SRC` into file `DEST`
+| `cp -r SRC DEST`		| Copy directory `SRC` into directory `DEST`
+| `rm FILE`				| Remove (delete) file `DIR`
+| `rm -r DIR`			| Remove (delete) directory `DIR`
+| `mv SRC DEST`         | Move the file (or directory) `SRC` into the directory `DEST`.
 
-Then do part 6 of the shell lesson.  
+In this part of the lesson we'll explore
+and familiarize ourselves with such commands.
 
-For a more advanced audience, just do part 6, so that you can use 
-the script later in the workshop.  
\ No newline at end of file
+When you login to the cluster,
+you begin at the home directory:
+
+~~~
+[nelle@login001 ~]$ pwd
+~~~
+{: .bash}
+
+~~~
+/home/nelle
+~~~
+{: .output}
+
+You can list the files and folders
+in this directory using `ls`:
+
+~~~
+[nelle@login001 ~]$ ls
+~~~
+{: .bash}
+
+~~~
+genomics-workshop   genomics-workshop.zip
+~~~
+{: .output}
+
+> ## File and folder names
+> Notice that we used a hyphen (`-`) between the words
+> `genomics` and `workshop` for the folder `genomics-workshop`.
+> Always avoid using spaces in file and folder names.
+> To see why it's a bad idea, try running the following command:
+>
+> ~~~
+> [nelle@login001 ~]$ mkdir genomics workshop
+> ~~~
+>
+> Now, type `ls`. What do you see?
+>
+{: .callout}
+
+The `ls` command accepts many **switches**
```

Comment:
Make a note that these are sometimes called "flags" or "options".

Reply:
I second this!